scripts/dev/mcp-resources-probe-server.py

```
@@ -0,0 +1,292 @@
+        protocol_version = "HTTP/1.1"
+
+        def log_message(self, *a):
+            pass
+
+        def _headers_subset(self):
+            keep = ("mcp-session-id", "mcp-protocol-version", "accept", "content-type", "last-event-id")
+            return {k: v for k, v in self.headers.items() if k.lower() in keep}
+
+        def do_POST(self):
+            if self.path.rstrip("/") != "/mcp":
+                self.send_response(404); self.end_headers(); return
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)